src/org/usfirst/frc/team4571/robot/commands/MillCommand.java

```diff
@@ -0,0 +1,48 @@
+package org.usfirst.frc.team4571.robot.commands;
+
+import org.usfirst.frc.team4571.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/**
+ *
+ */
+public class MillCommand extends Command {
+
+    public MillCommand() {
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    	requires(Robot.MILL_SUBSYSTEM);
+    }
+    public void forward(){
+    	Robot.MILL_SUBSYSTEM.forward(SmartDashboard.getDouble("MillForwardSpeed", 0.5));
+    }
+    public void reverse(){
+    	Robot.MILL_SUBSYSTEM.reverse(SmartDashboard.getDouble("MillReverseSpeed", -0.5));
+    }
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
```

Comment:
I'm sorry, but I don't quite understand what you're trying to say. Can you please explain this part a little more; and what is "the reverse of stop method"?